tools/segment/src/main/scala/org/allenai/nlpstack/segment/StanfordSegmenter.scala

```diff
@@ -0,0 +1,34 @@
+package org.allenai.nlpstack.segment
+
+import java.util.Properties
+
+import edu.stanford.nlp.ling.CoreAnnotations
+
+import scala.collection.JavaConverters._
+
+import edu.stanford.nlp.ling.CoreAnnotations.SentencesAnnotation
+import edu.stanford.nlp.pipeline.{ Annotation, StanfordCoreNLP }
+import org.allenai.nlpstack.core.{ Segment, Segmenter }
+
+object StanfordSegmenter extends Segmenter {
+  /* This is a bit unfortunate. In Stanford, you tokenize first, and then
+   * segment. In nlpstack, it's the other way around. We solve the problem by
+   * tokenizing twice, once here to get the sentences, and then again in
+   * StanfordTokenizer. */
+
+  private val pipeline = {
```

Comment:
I was making these lazy in ermine, to delay initialization until use. I'm not sure that was the right choice.

Does each `StanfordCoreNLP` instance load its own copy of the models &c that it needs?